scenarioo-client/app/components/detailarea/detailarea.component.css

```diff
@@ -20,42 +25,57 @@ sc-detailarea button.btn.btn-primary:active {
     color: #333333;
 }
 
+/* General flex box layout styles */
 
-/* Detailarea */
+sc-detailarea .flex {
+    display: flex;
+}
 
-sc-detailarea .sc-meta-data-panel {
-    margin-top: 35px;
+sc-detailarea .flex-fill {
+    flex: 1 1 auto;
 }
 
-sc-detailarea .detail-container {
+/* Colapsable Details Panel */
+
+sc-detailarea .details-panel {
     border: 1px solid #ccc;
     border-right: 0;
-    display: table;
     padding: unset;
     background-color: #eeeeee;
-    right: 0;
-}
-
-sc-detailarea .detail-row {
-    display: table-row;
 }
 
-sc-detailarea .detail-cell {
-    display: table-cell;
-}
-
-sc-detailarea .detail-button {
+sc-detailarea .details-collapse-button {
     background-color: #CCCCCC;
     width: 30px;
+    min-width: 30px; /* do never shrink */
     font-size: x-large;
     color: #337AB7;
     cursor: pointer;
 }
 
+sc-detailarea .sticky-arrow-container {
+    display: block;
+    vertical-align: top;
+    top: 0;
+    height: 100%;
+}
+
+sc-detailarea .sticky-arrow-icon-area {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 33vh; /* 2 times this plus the height should add to 100vh => sticky arrow is centered once full container is scrolled into view */
+    height: 34vh;
+    /* reasons for choosen height - it seems a good compromise between:
+     * 100vh = having to scroll too far down on small screens until you see the arrow
+     * 5vh = having the arrow too near to the top of the details panel until full panel scrolled into view
+     */
```

Comment:
What about something like this:
`position: fixed;`
`top: 50vh;`

Reply:
nope, this is not what I want - the problem here is that it with fixed it can scroll out of the container box - with sticky it doesnt.

Reply:
but i improved the positioning again - now that the size of the panel is at least the whole page